MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m

```diff
@@ -245,21 +245,14 @@ + (MSAppleErrorLog *)errorLogFromCrashReport:(MSPLCrashReport *)report {
   // errorLog.architecture is an optional. The Android SDK will set it while for iOS, the file will be set
   // server-side using primaryArchitectureId and architectureVariantId.
 
-  // HockeyApp didn't use report.exceptionInfo for this field but exception.name in case of an unhandled exception or
-  // the report.signalInfo.name
-  // more so, for BITCrashDetails, we used the exceptionInfo.exceptionName for a field called exceptionName. FYI: Gwynne
-  // has no idea. Andreas will be next ;)
-  errorLog.osExceptionType = report.exceptionInfo.exceptionName ?: report.signalInfo.name;
-
+  errorLog.osExceptionType = report.signalInfo.name;
   errorLog.osExceptionCode = report.signalInfo.code;
-
   errorLog.osExceptionAddress =
   [NSString stringWithFormat:@"0x%" PRIx64, report.signalInfo.address];
 
   errorLog.exceptionReason =
   [self extractExceptionReasonFromReport:report ofCrashedThread:crashedThread is64bit:is64bit];
-
-  errorLog.exceptionType = report.signalInfo.name;
+  errorLog.exceptionType = report.hasExceptionInfo ? report.exceptionInfo.exceptionName : nil;
```

Comment:
Name of signal was writed in `osExceptionType`. And `exceptionType` without exception must be `nil` (See comment to the field).